### ISSUE TITLE
Prototype 2: Momma Cub Club! Mobile

### ISSUE THREAD

**Issue by SylvainTran** [SylvainTran/cart315_w2021_prototype2#1]
The interest in this project was to create a casual lighthearted themed game. Accessibility, and mobile-oriented gameplay were the focus. This is a "big picture" and technical prototype, not a level design prototype. The arching game genre is the management simulator.

## Questions for playtesters:
Itch.io link: https://realsylvaintran.itch.io/momma-cub-club

Note: For now, you can only interact with the Training Centre and the Cub Buyers buildings, and while you can sell cubs, you can't currently buy more so there is no way to sustain the game yet. The clock works but there aren't any in-game events yet.

Considering that other features are incomplete : 

0. What is your opinion on the choice of the player's POV, atmosphere (music, art style), theme and direction of the gameplay (from what you've deduced so far)? 
1. What do you think about having to interact/click different buildings in order to do things in-game?
2. Should the gameplay around training the cubs be more engaging at the training centre? [Imagine that cubs' stats mattered in a future update] Would the statistics UI information at the bottom (when clicking on a cub) useful in that case and is it presented clearly?
3. Do you envision these buildings' options to have a UI graphical menu that detail the different actions that exist for them or mini-games like at the Training Centre's?
4. How would you feel if the buildings' hypothetical UI were the only way to execute actions, as opposed to how it is now? Comment on the UI that you would feel is most intuitive in this game.
5. What are the gameplay features that you wish you could try the most given the current state of the prototype?
6. What kinds of in-game events would you want to see naturally occur in the in-game calendar?
7. What is your opinion on the role of the player in the naturally occurring economy cycle in-game? Should the player initiate his own trading and purchase of economical goods more, or should they occur independently of the player?
8. Do you enjoy grabbing the cubs and dropping them? Are the FX interesting for that purpose--do they make you feel that a trivial action can be rewarding?
9. Would you want to see more gameplay related to cub grabbing and dropping, in the context of the different buildings' allowed gameplay actions?
10. Would you play this on MOBILE (e.g., your smartphone)?
11. Do you see an accessible and playful management game emerging from this prototype, and do you feel it would be interesting to progress further in the game given the current game loop (see point #3 in the Tutorial in main menu or above)? Finally, if I added the Slaughter House feature, would you continue playing the game? Thanks for playing!

**Comment by Naomi-Ifergan** [comment]
**0. What is your opinion on the choice of the player's POV, atmosphere (music, art style), theme and direction of the gameplay (from what you've deduced so far)?**

The POV at the start of the game (before clicking on a building) is kind of weird. Other than that the POV during the game itself is fine. I enjoyed the music, it’s a perfect fit for the atmosphere. The visuals are nice, however I wished the visuals of the game itself matched the start game page because the colors are different other than that everything looks and sounds good!

**1. What do you think about having to interact/click different buildings in order to do things in-game?**

I really like that feature! I like how it zooms into the building the user clicked on. 

**2. Should the gameplay around training the cubs be more engaging at the training centre? [Imagine that cubs' stats mattered in a future update] Would the statistics UI information at the bottom (when clicking on a cub) useful in that case and is it presented clearly?**

I feel like it should be more challenging for the user to engage at the training center. Right now I just waited for the cubs to go enter their little resting area on their own. SO maybe have the player drag them into place. And while the gate is opened the animals seem to be trapped even though the gate is opened. But basically if the gate is still opened, the animals should be able to exit it to challenge the user, maybe allow the animals to leave while the gate is opened. This would give the user a hard time to keep the animals inside while trying to bring the other animals inside. 

**3. Do you envision these buildings' options to have a UI graphical menu that detail the different actions that exist for them or mini-games like at the Training Centre's?**

Yes. I think it would be helpful.

**4. How would you feel if the buildings' hypothetical UI were the only way to execute actions, as opposed to how it is now? Comment on the UI that you would feel is most intuitive in this game.**

I really like the fact that I can choose which building to engage with whenever I want. Rather than having an order in which I’d have to respect such as first going to the training center and so on. I think it’s nice that I have the ability to pick where I want to start the game and where I want to go next. 

**5. What are the gameplay features that you wish you could try the most given the current state of the prototype**?

I would have liked to see how the Slaughter House would work. 

**6. What kinds of in-game events would you want to see naturally occur in the in-game calendar?**

That’s a good question but I’m really not sure.

**7. What is your opinion on the role of the player in the naturally occurring economy cycle in-game? Should the player initiate his own trading and purchase of economical goods more, or should they occur independently of the player?**

Yes the player should be able to initiate his own trading.

**8. Do you enjoy grabbing the cubs and dropping them? Are the FX interesting for that purpose--do they make you feel that a trivial action can be rewarding?**

The drag and drop functions are fun to play with!

**9. Would you want to see more gameplay related to cub grabbing and dropping, in the context of the different buildings' allowed gameplay actions?**

Yes for sure!

**10. Would you play this on MOBILE (e.g., your smartphone)?**

I can definitely see it as a mobile game however it is not something I would play myself because it’s not the type of game I actually play.

**11. Do you see an accessible and playful management game emerging from this prototype, and do you feel it would be interesting to progress further in the game given the current game loop (see point #3 in the Tutorial in main menu or above)? Finally, if I added the Slaughter House feature, would you continue playing the game? Thanks for playing!**

I definitely see it’s potential and given the fact that this was only a prototype I’m excited to see it as a finished game. You always go above and beyond in all your games as I’ve seen over the past few semesters and even though it’s not the type of game I would usually play, I am curious to see it as a finished game. As for the Slaughter House, some might say it will be disturbing but I wouldn’t mind it, it all depends how you make it work in regards to the visuals and so on.

**Comment by marafrass** [comment]
- What is your opinion on the choice of the player's POV, atmosphere (music, art style), theme and direction of the gameplay (from what you've deduced so far)?
I dig it! The music can get a little repetitive over time, but I think the tone fits well for this kind of game and gives you an idea of what the final version would be like. 
- What do you think about having to interact/click different buildings in order to do things in-game?
This adds to my answer to the first question as well - I think it works really well and gives you a clear idea of your operation, and makes you feel like you're running some kind of company/training camp/slaughter...house? It's also very intuitive and easy to grasp.
- Should the gameplay around training the cubs be more engaging at the training centre? [Imagine that cubs' stats mattered in a future update] Would the statistics UI information at the bottom (when clicking on a cub) useful in that case and is it presented clearly?
I think so, yeah - I was unsure if placing the cubs in the training pen actually did anything at first. The "level up" effect seemed pretty similar to the one you get just by picking up the cubs, so I was unsure if it was working correctly. I was also unable to actually sell any of the cubs, but that might not be implemented yet or I mightve misunderstood how it worked.
- Do you envision these buildings' options to have a UI graphical menu that detail the different actions that exist for them or mini-games like at the Training Centre's?
I'd say so, yes. It'd be nice to have a general theme or position for all UI in the buildings, so knowledge of UI carries from one activity to another.
- How would you feel if the buildings' hypothetical UI were the only way to execute actions, as opposed to how it is now? Comment on the UI that you would feel is most intuitive in this game.
I feel that it'd be doable, but using them would require a lot of visual feedback, like outlines or other effects, as well as sound.
- What are the gameplay features that you wish you could try the most given the current state of the prototype?
I think I'd prefer starting with a single cub to focus on, both to learn the ropes and to actually consider if each of them have value or if it's just a big batch. Beyond that it'd be nice to make more meaningful choices about how you train your cubs, and consider their own strengths vs what the market desires, for example.
- What kinds of in-game events would you want to see naturally occur in the in-game calendar?
Stray cubs arriving at the compound, arguments between cubs - small narrative events, similar to CK2/3, where you calculate risk/reward and what you want the future of that/those cubs to be.
- What is your opinion on the role of the player in the naturally occurring economy cycle in-game? Should the player initiate his own trading and purchase of economical goods more, or should they occur independently of the player?
I think it's more interesting if the economy exists outside of the player, and they have to adapt to it. Otherwise, the player could basically control the economy and make sure they always profit. Some layer of RNG would be nice, or even a competitive AI trader!
- Do you enjoy grabbing the cubs and dropping them? Are the FX interesting for that purpose--do they make you feel that a trivial action can be rewarding?
Noises and VFX always helps! Right now, I think they were looping which got a little repetitive, but I think you're on the right track. I'd also love to see more numbers go up!
- Would you want to see more gameplay related to cub grabbing and dropping, in the context of the different buildings' allowed gameplay actions?
I dont think so - I think it's fine as a general management tool and to make that work feel entertaining, but I wouldnt design gameplay around it.
- Would you play this on MOBILE (e.g., your smartphone)?
I play like one or two puzzle games on my phone and nothing else, so not really - I could see this type of game working there, though.
- Do you see an accessible and playful management game emerging from this prototype, and do you feel it would be interesting to progress further in the game given the current game loop (see point #3 in the Tutorial in main menu or above)? Finally, if I added the Slaughter House feature, would you continue playing the game? Thanks for playing!
I think there's definite potential! I'd love to see how it all fits into a bigger world or a bigger economy, just to make it a little more challenging and to make the decisions matter a bit more. Would be hyped to see where you'd take this, even if it isnt my type of game!

**Comment by amandaclement** [comment]
**What is your opinion on the choice of the player's POV, atmosphere (music, art style), theme and direction of the gameplay (from what you've deduced so far)?**
It's quite charming and lively. I do agree with Noami's comment that the start page (mainly the instruction text) doesn't really match the rest of the visuals but besides that, it all works well! The environment looks great.

**What do you think about having to interact/click different buildings in order to do things in-game?**
I'm a fan of it! Helps the player get a clear idea of where they're heading. The zoom effect is also very satisfying.

**Should the gameplay around training the cubs be more engaging at the training centre? [Imagine that cubs' stats mattered in a future update] Would the statistics UI information at the bottom (when clicking on a cub) useful in that case and is it presented clearly?**
Yes, I think it would be useful and would help to make the game feel more engaging. The information is presented clearly however the process of training the cubs was not entirely clear to me. I wasn't sure if I was playing it correctly so it may be helpful to have a bit more guidance there (maybe when the player first picks up a cub, have arrows/text to indicate where to drag and what to do).

**Do you envision these buildings' options to have a UI graphical menu that detail the different actions that exist for them or mini-games like at the Training Centre's?**
I would picture this game to evolve into a collection of mini games like at the Training Centre. 

**How would you feel if the buildings' hypothetical UI were the only way to execute actions, as opposed to how it is now? Comment on the UI that you would feel is most intuitive in this game.**
I think it would really depend on the execution and level of detail. Could possibly be confusing for the player?

**What are the gameplay features that you wish you could try the most given the current state of the prototype?**
Really curious about the Slaughter House feature! I would also be interested in seeing how the Training Centre could be further developed.

**What kinds of in-game events would you want to see naturally occur in the in-game calendar?**
Would be nice to have a wider variety of cubs, or introduce/remove (kill them off? they escape?) as you play.

**What is your opinion on the role of the player in the naturally occurring economy cycle in-game? Should the player initiate his own trading and purchase of economical goods more, or should they occur independently of the player?**
Preferably a mix of both. I think it would be best if the player could initiate their own trading within the pre-determined limits of the economy.

**Do you enjoy grabbing the cubs and dropping them? Are the FX interesting for that purpose--do they make you feel that a trivial action can be rewarding?**
Yes, I think you did a great job with that! The visual and sound effects you implemented add energy to the game.

**Would you want to see more gameplay related to cub grabbing and dropping, in the context of the different buildings' allowed gameplay actions?**
I wouldn't say it's necessary but I also wouldn't object to it.

**Would you play this on MOBILE (e.g., your smartphone)?**
I don't really play games on mobile but it seems well-suited for it!

**Do you see an accessible and playful management game emerging from this prototype, and do you feel it would be interesting to progress further in the game given the current game loop (see point #3 in the Tutorial in main menu or above)? Finally, if I added the Slaughter House feature, would you continue playing the game?** 
Yes, I can definitely see a fuller game evolving from this prototype. I'd love to see how you further develop the pre-existing game elements and how you implement the features you had planned out but haven't yet executed. And yes, the Slaughter House sounds intriguing!

**Comment by SylvainTran** [comment]
> **0. What is your opinion on the choice of the player's POV, atmosphere (music, art style), theme and direction of the gameplay (from what you've deduced so far)?**
> 
> The POV at the start of the game (before clicking on a building) is kind of weird. Other than that the POV during the game itself is fine. I enjoyed the music, it’s a perfect fit for the atmosphere. The visuals are nice, however I wished the visuals of the game itself matched the start game page because the colors are different other than that everything looks and sounds good!
> 

Gotcha, I will make sure the visuals match the start game page. I was trying to see if a weather system would be interesting. 

> **1. What do you think about having to interact/click different buildings in order to do things in-game?**
> 
> I really like that feature! I like how it zooms into the building the user clicked on.

OK.

> 
> **2. Should the gameplay around training the cubs be more engaging at the training centre? [Imagine that cubs' stats mattered in a future update] Would the statistics UI information at the bottom (when clicking on a cub) useful in that case and is it presented clearly?**
> 
> I feel like it should be more challenging for the user to engage at the training center. Right now I just waited for the cubs to go enter their little resting area on their own. SO maybe have the player drag them into place. And while the gate is opened the animals seem to be trapped even though the gate is opened. But basically if the gate is still opened, the animals should be able to exit it to challenge the user, maybe allow the animals to leave while the gate is opened. This would give the user a hard time to keep the animals inside while trying to bring the other animals inside.
> 

You're right, I didn't get enough time to add some challenge at the Training Centre. I will note what you suggested.

> **3. Do you envision these buildings' options to have a UI graphical menu that detail the different actions that exist for them or mini-games like at the Training Centre's?**
> 
> Yes. I think it would be helpful.

OK.

> 
> **4. How would you feel if the buildings' hypothetical UI were the only way to execute actions, as opposed to how it is now? Comment on the UI that you would feel is most intuitive in this game.**
> 
> I really like the fact that I can choose which building to engage with whenever I want. Rather than having an order in which I’d have to respect such as first going to the training center and so on. I think it’s nice that I have the ability to pick where I want to start the game and where I want to go next.
> 

OK.

> **5. What are the gameplay features that you wish you could try the most given the current state of the prototype**?
> 
> I would have liked to see how the Slaughter House would work.

The slaughterhouse is my top priority.

> 
> **6. What kinds of in-game events would you want to see naturally occur in the in-game calendar?**
> 
> That’s a good question but I’m really not sure.

OK.

> 
> **7. What is your opinion on the role of the player in the naturally occurring economy cycle in-game? Should the player initiate his own trading and purchase of economical goods more, or should they occur independently of the player?**
> 
> Yes the player should be able to initiate his own trading.

OK.

> 
> **8. Do you enjoy grabbing the cubs and dropping them? Are the FX interesting for that purpose--do they make you feel that a trivial action can be rewarding?**
> 
> The drag and drop functions are fun to play with!
> 

OK.

> **9. Would you want to see more gameplay related to cub grabbing and dropping, in the context of the different buildings' allowed gameplay actions?**
> 
> Yes for sure!

OK.

> 
> **10. Would you play this on MOBILE (e.g., your smartphone)?**
> 
> I can definitely see it as a mobile game however it is not something I would play myself because it’s not the type of game I actually play.

I see.

> 
> **11. Do you see an accessible and playful management game emerging from this prototype, and do you feel it would be interesting to progress further in the game given the current game loop (see point #3 in the Tutorial in main menu or above)? Finally, if I added the Slaughter House feature, would you continue playing the game? Thanks for playing!**
> 
> I definitely see it’s potential and given the fact that this was only a prototype I’m excited to see it as a finished game. You always go above and beyond in all your games as I’ve seen over the past few semesters and even though it’s not the type of game I would usually play, I am curious to see it as a finished game. As for the Slaughter House, some might say it will be disturbing but I wouldn’t mind it, it all depends how you make it work in regards to the visuals and so on.

Thanks, I appreciate the comment!

**Comment by SylvainTran** [comment]
> * What is your opinion on the choice of the player's POV, atmosphere (music, art style), theme and direction of the gameplay (from what you've deduced so far)?
>   I dig it! The music can get a little repetitive over time, but I think the tone fits well for this kind of game and gives you an idea of what the final version would be like.

Sounds good!

> * What do you think about having to interact/click different buildings in order to do things in-game?
>   This adds to my answer to the first question as well - I think it works really well and gives you a clear idea of your operation, and makes you feel like you're running some kind of company/training camp/slaughter...house? It's also very intuitive and easy to grasp.

Cool, I will keep this feature then.

> * Should the gameplay around training the cubs be more engaging at the training centre? [Imagine that cubs' stats mattered in a future update] Would the statistics UI information at the bottom (when clicking on a cub) useful in that case and is it presented clearly?
>   I think so, yeah - I was unsure if placing the cubs in the training pen actually did anything at first. The "level up" effect seemed pretty similar to the one you get just by picking up the cubs, so I was unsure if it was working correctly. I was also unable to actually sell any of the cubs, but that might not be implemented yet or I mightve misunderstood how it worked.

You're right, I will put some time on this. The feedback needs to be clear at every stage. I don't know why you couldn't sell any cubs, but I will take note of a possible bug.

> * Do you envision these buildings' options to have a UI graphical menu that detail the different actions that exist for them or mini-games like at the Training Centre's?
>   I'd say so, yes. It'd be nice to have a general theme or position for all UI in the buildings, so knowledge of UI carries from one activity to another.

OK.

> * How would you feel if the buildings' hypothetical UI were the only way to execute actions, as opposed to how it is now? Comment on the UI that you would feel is most intuitive in this game.
>   I feel that it'd be doable, but using them would require a lot of visual feedback, like outlines or other effects, as well as sound.

Sounds like a lot of work.

> * What are the gameplay features that you wish you could try the most given the current state of the prototype?
>   I think I'd prefer starting with a single cub to focus on, both to learn the ropes and to actually consider if each of them have value or if it's just a big batch. Beyond that it'd be nice to make more meaningful choices about how you train your cubs, and consider their own strengths vs what the market desires, for example.

Makes sense. I started with 10 to show a bit of what would happen in the game. I will take note of easing things up more progressively as far as tutorials would require, like you suggested.

> * What kinds of in-game events would you want to see naturally occur in the in-game calendar?
>   Stray cubs arriving at the compound, arguments between cubs - small narrative events, similar to CK2/3, where you calculate risk/reward and what you want the future of that/those cubs to be.

Great ideas.

> * What is your opinion on the role of the player in the naturally occurring economy cycle in-game? Should the player initiate his own trading and purchase of economical goods more, or should they occur independently of the player?
>   I think it's more interesting if the economy exists outside of the player, and they have to adapt to it. Otherwise, the player could basically control the economy and make sure they always profit. Some layer of RNG would be nice, or even a competitive AI trader!

You're right. The AI trader and outside economy is definitely on the list.

> * Do you enjoy grabbing the cubs and dropping them? Are the FX interesting for that purpose--do they make you feel that a trivial action can be rewarding?
>   Noises and VFX always helps! Right now, I think they were looping which got a little repetitive, but I think you're on the right track. I'd also love to see more numbers go up!

OK. Yeah, numbers sound good.

> * Would you want to see more gameplay related to cub grabbing and dropping, in the context of the different buildings' allowed gameplay actions?
>   I dont think so - I think it's fine as a general management tool and to make that work feel entertaining, but I wouldnt design gameplay around it.

OK.

> * Would you play this on MOBILE (e.g., your smartphone)?
>   I play like one or two puzzle games on my phone and nothing else, so not really - I could see this type of game working there, though.

Haha, gotcha.

> * Do you see an accessible and playful management game emerging from this prototype, and do you feel it would be interesting to progress further in the game given the current game loop (see point #3 in the Tutorial in main menu or above)? Finally, if I added the Slaughter House feature, would you continue playing the game? Thanks for playing!
>   I think there's definite potential! I'd love to see how it all fits into a bigger world or a bigger economy, just to make it a little more challenging and to make the decisions matter a bit more. Would be hyped to see where you'd take this, even if it isnt my type of game!

A larger world and economy is definitely on the wishlist. Thanks for your time!

**Comment by SylvainTran** [comment]
> **What is your opinion on the choice of the player's POV, atmosphere (music, art style), theme and direction of the gameplay (from what you've deduced so far)?**
> It's quite charming and lively. I do agree with Noami's comment that the start page (mainly the instruction text) doesn't really match the rest of the visuals but besides that, it all works well! The environment looks great.
> 

Thanks! It's all free stuff on the asset store. I'll make sure that final assets (if I develop this game further) stay close to these.

> **What do you think about having to interact/click different buildings in order to do things in-game?**
> I'm a fan of it! Helps the player get a clear idea of where they're heading. The zoom effect is also very satisfying.
> 

OK, I will keep this feature and polish it.

> **Should the gameplay around training the cubs be more engaging at the training centre? [Imagine that cubs' stats mattered in a future update] Would the statistics UI information at the bottom (when clicking on a cub) useful in that case and is it presented clearly?**
> Yes, I think it would be useful and would help to make the game feel more engaging. The information is presented clearly however the process of training the cubs was not entirely clear to me. I wasn't sure if I was playing it correctly so it may be helpful to have a bit more guidance there (maybe when the player first picks up a cub, have arrows/text to indicate where to drag and what to do).
> 

You're right, it isn't clear to me either. I will look into making the process more intuitive and show feedback/guidance step-by-step as you suggested.

> **Do you envision these buildings' options to have a UI graphical menu that detail the different actions that exist for them or mini-games like at the Training Centre's?**
> I would picture this game to evolve into a collection of mini games like at the Training Centre.
> 

Cool! I like the idea of mini-games, glad we're on the same page again.

> **How would you feel if the buildings' hypothetical UI were the only way to execute actions, as opposed to how it is now? Comment on the UI that you would feel is most intuitive in this game.**
> I think it would really depend on the execution and level of detail. Could possibly be confusing for the player?
> 

Yes, given my inexperience doing UIs, this may prove to be tedious, so I'm more in favour of mini-games with guided steps instead.

> **What are the gameplay features that you wish you could try the most given the current state of the prototype?**
> Really curious about the Slaughter House feature! I would also be interested in seeing how the Training Centre could be further developed.
> 

The slaughterhouse is definitely my next priority. It should be "bloody glorious", hopefully in a fun way. Yes, I will also develop the Training Centre further with what was suggested. It might be very different the next time.

> **What kinds of in-game events would you want to see naturally occur in the in-game calendar?**
> Would be nice to have a wider variety of cubs, or introduce/remove (kill them off? they escape?) as you play.
> 

Great ideas.

> **What is your opinion on the role of the player in the naturally occurring economy cycle in-game? Should the player initiate his own trading and purchase of economical goods more, or should they occur independently of the player?**
> Preferably a mix of both. I think it would be best if the player could initiate their own trading within the pre-determined limits of the economy.
> 

As expected. Yes

> **Do you enjoy grabbing the cubs and dropping them? Are the FX interesting for that purpose--do they make you feel that a trivial action can be rewarding?**
> Yes, I think you did a great job with that! The visual and sound effects you implemented add energy to the game.
> 

Thanks.

> **Would you want to see more gameplay related to cub grabbing and dropping, in the context of the different buildings' allowed gameplay actions?**
> I wouldn't say it's necessary but I also wouldn't object to it.
> 

OK.

> **Would you play this on MOBILE (e.g., your smartphone)?**
> I don't really play games on mobile but it seems well-suited for it!
> 

OK. Me neither. Yes, the drag mechanics carry over well with touch-based input. I just need to think about relevant ways of using these in the game.

> **Do you see an accessible and playful management game emerging from this prototype, and do you feel it would be interesting to progress further in the game given the current game loop (see point #3 in the Tutorial in main menu or above)? Finally, if I added the Slaughter House feature, would you continue playing the game?**
> Yes, I can definitely see a fuller game evolving from this prototype. I'd love to see how you further develop the pre-existing game elements and how you implement the features you had planned out but haven't yet executed. And yes, the Slaughter House sounds intriguing!

Thanks, it's definitely encouraging!

**Comment by SylvainTran** [comment]
@Naomi-Ifergan @marafrass @amandaclement 's feedbacks were addressed and taken into consideration. Thanks for your time, and sorry for making you work through 12 questions 👍🏼 . I will tag you again if I have a question about what you raised up. Still waiting on some other people, but it's great so far. Huge appreciations.

**Comment by fredlabbe** [comment]
What is your opinion on the choice of the player's POV, atmosphere (music, art style), theme and direction of the gameplay (from what you've deduced so far)? 
I like it! Childish, but fun and relaxing. Useful to get away from our sad confined everyday lives.

What do you think about having to interact/click different buildings in order to do things in-game? 
Love it! Just like an RTS, which are some of my favorite types of games.

Should the gameplay around training the cubs be more engaging at the training centre? [Imagine that cubs' stats mattered in a future update] Would the statistics UI information at the bottom (when clicking on a cub) useful in that case and is it presented clearly? 
It's already quite complicated, so it should be done clearly and with good graphics so it's not overwealming, but yeah. 

Do you envision these buildings' options to have a UI graphical menu that detail the different actions that exist for them or mini-games like at the Training Centre's? 
Could be a good idea, when you hover on them they could indicate what they do or are used for.

How would you feel if the buildings' hypothetical UI were the only way to execute actions, as opposed to how it is now? Comment on the UI that you would feel is most intuitive in this game. 
I like it how it is right now, clicking on them makes it clear and defined that you are at different places that do different things.

What are the gameplay features that you wish you could try the most given the current state of the prototype? 
I would personally love to be able to create/add more buildings myself when I have enough money. 

What kinds of in-game events would you want to see naturally occur in the in-game calendar? 
Weather challenges, wolves attacking, economic crisis or wtv could be good ideas!

What is your opinion on the role of the player in the naturally occurring economy cycle in-game? Should the player initiate his own trading and purchase of economical goods more, or should they occur independently of the player? 
It depends on what you want your game to be. Not all players enjoy spending too much time on economies usually, so you should keep that in mind in your decisions.

Do you enjoy grabbing the cubs and dropping them? Are the FX interesting for that purpose--do they make you feel that a trivial action can be rewarding? 
They are interesting, but sometimes, when I try tio grab one and bring it into the training enclosure, it grabs one that is already there and they both get out. 

Would you want to see more gameplay related to cub grabbing and dropping, in the context of the different buildings' allowed gameplay actions? 
It could be nice, but maybe you could have an inventory or something? To bring more at once than only one.

Would you play this on MOBILE (e.g., your smartphone)? 
Yes, but it could be challenging since there are many things on the screen and aphone is quite small.

Do you see an accessible and playful management game emerging from this prototype, and do you feel it would be interesting to progress further in the game given the current game loop (see point #3 in the Tutorial in main menu or above)? 
Yes, I think it could be interesting. I'm not sure I would play it on my phone since I'm not a mobile gamer, but for people who do it could be interesting.

Finally, if I added the Slaughter House feature, would you continue playing the game? Thanks for playing! 
Personally, yes, but it could definitely be an issue with PETA or vegan groups. Try not to get canceled before your company gets huge!

**Comment by SylvainTran** [comment]
> What is your opinion on the choice of the player's POV, atmosphere (music, art style), theme and direction of the gameplay (from what you've deduced so far)?
> I like it! Childish, but fun and relaxing. Useful to get away from our sad confined everyday lives.
> 
Haha, cool.
> What do you think about having to interact/click different buildings in order to do things in-game?
> Love it! Just like an RTS, which are some of my favorite types of games.
> 
I love RTS too. Casual RTS is crazy.
> Should the gameplay around training the cubs be more engaging at the training centre? [Imagine that cubs' stats mattered in a future update] Would the statistics UI information at the bottom (when clicking on a cub) useful in that case and is it presented clearly?
> It's already quite complicated, so it should be done clearly and with good graphics so it's not overwealming, but yeah.
> 
OK, got it. For the dev it's always simpler.
> Do you envision these buildings' options to have a UI graphical menu that detail the different actions that exist for them or mini-games like at the Training Centre's?
> Could be a good idea, when you hover on them they could indicate what they do or are used for.
> 
OK.
> How would you feel if the buildings' hypothetical UI were the only way to execute actions, as opposed to how it is now? Comment on the UI that you would feel is most intuitive in this game.
> I like it how it is right now, clicking on them makes it clear and defined that you are at different places that do different things.
> 
Glad to hear this worked.
> What are the gameplay features that you wish you could try the most given the current state of the prototype?
> I would personally love to be able to create/add more buildings myself when I have enough money.
> 
Noted
> What kinds of in-game events would you want to see naturally occur in the in-game calendar?
> Weather challenges, wolves attacking, economic crisis or wtv could be good ideas!
> 
Fun ideas! 
> What is your opinion on the role of the player in the naturally occurring economy cycle in-game? Should the player initiate his own trading and purchase of economical goods more, or should they occur independently of the player?
> It depends on what you want your game to be. Not all players enjoy spending too much time on economies usually, so you should keep that in mind in your decisions.
> 
That's true... I'm biased.
> Do you enjoy grabbing the cubs and dropping them? Are the FX interesting for that purpose--do they make you feel that a trivial action can be rewarding?
> They are interesting, but sometimes, when I try tio grab one and bring it into the training enclosure, it grabs one that is already there and they both get out.
> 
Hmm, I think I see what you mean.
> Would you want to see more gameplay related to cub grabbing and dropping, in the context of the different buildings' allowed gameplay actions?
> It could be nice, but maybe you could have an inventory or something? To bring more at once than only one.
> 
Interesting idea, noted.
> Would you play this on MOBILE (e.g., your smartphone)?
> Yes, but it could be challenging since there are many things on the screen and aphone is quite small.
> 
OK yeah, true.
> Do you see an accessible and playful management game emerging from this prototype, and do you feel it would be interesting to progress further in the game given the current game loop (see point #3 in the Tutorial in main menu or above)?
> Yes, I think it could be interesting. I'm not sure I would play it on my phone since I'm not a mobile gamer, but for people who do it could be interesting.
> 
Cool.
> Finally, if I added the Slaughter House feature, would you continue playing the game? Thanks for playing!
> Personally, yes, but it could definitely be an issue with PETA or vegan groups. Try not to get canceled before your company gets huge!
Hahaha that's definitely something to consider.
